app/views/individual/claimenrolment/ClaimEnrolmentConfirmation.scala.html

```diff
@@ -62,10 +62,8 @@ <h2 class="govuk-heading-m">@messages("claimConfirm.heading")</h2>
                     </a>))
             </p>
 
-            <p class = "govuk-body">@messages("sign-up-complete.whatHappensNow.para2")</p>
-            <br>
+            <p class="govuk-body govuk-!-margin-bottom-6">@messages("sign-up-complete.whatHappensNow.para2")</p>
             @buttonHelper(messages("claimConfirm.continueButton"))
```

Comment:
Without `<br>` the sign out link gets displayed on the same line as the continue button:
```suggestion
            @buttonHelper(messages("claimConfirm.continueButton"), classes = Some("govuk-!-display-block"))
```